modules/space/rendering/renderablestars.cpp

```diff
@@ -533,100 +458,95 @@ RenderableStars::RenderableStars(const ghoul::Dictionary& dictionary)
     )
     , _fixedColor(FixedColorInfo, glm::vec3(1.f), glm::vec3(0.f), glm::vec3(1.f))
     , _filterOutOfRange(FilterOutOfRangeInfo, false)
-    , _pointSpreadFunctionTexturePath(PsfTextureInfo)
-    , _psfMethodOption(
-        PSFMethodOptionInfo,
-        properties::OptionProperty::DisplayType::Dropdown
-    )
-    , _psfMultiplyOption(
-        SizeCompositionOptionInfo,
-        properties::OptionProperty::DisplayType::Dropdown
-    )
-    , _lumCent(LumPercentInfo, 0.5f, 0.f, 3.f)
-    , _radiusCent(RadiusPercentInfo, 0.5f, 0.f, 3.f)
-    , _brightnessCent(BrightnessPercentInfo, 0.5f, 0.f, 3.f)
+    , _halo{
+        properties::PropertyOwner(HaloOwnerInfo),
+        properties::StringProperty(TextureInfo),
+        properties::FloatProperty(PartialOpacityInfo, 1.f, 0.f, 5.f),
+        properties::FloatProperty(ScaleInfo, 1.f, 0.f, 1.f)
+    }
+    , _glare{
+        properties::PropertyOwner(GlareOwnerInfo),
+        properties::StringProperty(TextureInfo),
+        properties::FloatProperty(PartialOpacityInfo, 1.f, 0.f, 5.f),
+        properties::FloatProperty(ScaleInfo, 1.f, 0.f, 1.f)
+    }
+    , _parameters{
+        properties::PropertyOwner(ParametersOwnerInfo),
+        properties::OptionProperty(
+            SizeCompositionOptionInfo,
+            properties::OptionProperty::DisplayType::Dropdown
+        ),
+        properties::FloatProperty(LumPercentInfo, 0.5f, 0.f, 3.f),
```

Comment:
These need some better documentation for when they are actually applied (i.e. that they come in effect depending on the size composition option). Also about what the "unit" for the value means, sort of

Reply:
Honestly, I have no idea what the units are there is a lot of mystical math happening in the shader.   If i knew that no user is actually using these, I'd be super happy to get rid of all other size composition options.

Reply:
Ah, ok.  Well that's a future thing then :)